NEWS.md

```diff
@@ -1,4 +1,4 @@
-# teal.osprey 0.1.14.9013
+# teal.osprey 0.1.14.9013.1
```

Comment:
hmm - should this have happened? @cicdguy

Reply:
No. That's weird. I'll look into it.

Reply:
Okay so this apparently came from pre-release when it existed. Not an active concern, just something that can be cleaned up here itself.

<img width="852" alt="Screen Shot 2022-08-25 at 6 17 11 AM" src="https://user-images.githubusercontent.com/26552821/186650666-84de41e6-4a7c-4e1d-88e7-fb2e1c8a83da.png">